spec.md

```diff
@@ -240,9 +240,10 @@ Some examples of _digests_ include the following:
 
 | digest                                                                  | description                |
 |-------------------------------------------------------------------------|----------------------------|
-| sha256:6c3c624b58dbbcd3c0dd82b4c53f04194d1247c6eebdaab7c610cf7d66709b3b | Common sha256 based digest |
+| `sha256:6c3c624b58dbbcd3c0dd82b4c53f04194d1247c6eebdaab7c610cf7d66709b3b` | Common sha256 based digest format |
 
-While the _algorithm_ does allow one to implement a wide variety of algorithms, compliant implementations SHOULD use sha256.
+Compliant implementations SHOULD use _algorithm_ of sha256 (until the ruled sufficiently broken).
```

Comment:
reword or remove the parenthetical statement

Reply:
removed. Since this will be captured in a version, it will stand as that version. No need to allude to removal.